addons/pos_self_order/views/pos_self_order.index.xml

```diff
@@ -9,6 +9,9 @@
                 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
                 <meta name="apple-mobile-web-app-capable" content="yes"/>
                 <meta name="mobile-web-app-capable" content="yes"/>
+                <link rel="manifest"  href="/pos-self/manifest" crossorigin="use-credentials"/>
```

Comment:
in this line, use the qweb templating system to render the link to the manifest.